pkg/interface/src/views/apps/chat/components/ChatMessage.tsx

```diff
@@ -445,8 +445,10 @@ export const Message = ({
                 </Box>
               );
             case 'mention':
+              const first = (i) => (i === 0);
               return (
                 <Mention
+                  marginLeft={first(i) ? "-4px" : "0"}
```

Comment:
Indigo would do this as just "1" I think. We could stick to theme variables.

(Alternatively — don't margins collapse? I wonder if we could avoid all this logic with clever margin stuff.)

Reply:
Unfortunately, margins only collapse vertically. From what I can discern, we want margins around mentions if they're surrounded by other content, and only on the right side if they're not. I will reverse the logic here and thread  to only positively apply margins.